charts/postgresql/values.yaml

```diff
@@ -15,7 +15,7 @@ global:
 image:
   registry: quay.io
   repository: astronomer/ap-postgresql
-  tag: 15.6.0-1
+  tag: 15.7.0-1
```

Comment:
There is currently no 15.7.0-1, only 15.7.0. https://quay.io/repository/astronomer/ap-postgresql?tab=tags

Reply:
https://github.com/astronomer/ap-vendor/blob/main/postgresql/version.txt if you can see here its 15.7.0-1 only

Reply:
Can I update this change ??

Reply:
Updated the requested change

Reply:
I know the reason why it has not published

Reply:
fixing it

Reply:
this is fixed now